spec/spec_helper.rb

```diff
@@ -21,25 +21,6 @@
 require 'mobility'
 require "mobility/backends/null"
 
-# Enable default plugins
-Mobility.configure do |config|
-  config.plugins do
-    backend
-    reader
-    writer
-    query
-    cache
-    dirty
-    fallbacks
-    presence
-    default
-    fallthrough_accessors
-    locale_accessors
-  end
-  config.plugin orm if orm
-  config.plugin :attribute_methods if defined?(ActiveRecord)
-end
```

Comment:
This is basically the point of this whole PR: to not set plugins globally in tests, but instead enable them selectively per test.